gandi/resource_glue_record.go

```diff
@@ -0,0 +1,173 @@
+package gandi
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"sort"
+	"time"
+
+	"github.com/go-gandi/go-gandi/domain"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceGlueRecord() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGlueRecordCreate,
+		Read:   resourceGlueRecordRead,
+		Update: resourceGlueRecordUpdate,
+		Delete: resourceGlueRecordDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"zone": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Domain name",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Host name of the glue record",
+			},
+			"values": {
+				Type:        schema.TypeList,
+				Required:    true,
+				ForceNew:    false,
```

Comment:
Useless since it is the default value, excepting if you really want to be explicit (up to you).

Reply:
Not a strong preference, removed.